dockstore-integration-testing/src/test/java/io/dockstore/client/cli/ExtendedTRSOpenApiIT.java

```diff
@@ -175,11 +176,16 @@ void testAggregatedMetricsPut() {
         extendedGa4GhApi.aggregatedMetricsPut(metrics, platform2, workflowId, workflowVersionId);
         workflow = workflowsApi.getPublishedWorkflow(workflow.getId(), "metrics");
         workflowVersion = workflow.getWorkflowVersions().stream().filter(v -> workflowVersionId.equals(v.getName())).findFirst().orElse(null);
+
         assertNotNull(workflowVersion);
         assertEquals(2, workflowVersion.getMetricsByPlatform().size(), "Version should have metrics for 2 platforms");
 
-        assertNotNull(workflowVersion.getMetricsByPlatform().get(platform1));
-        assertNotNull(workflowVersion.getMetricsByPlatform().get(platform2));
+        platform1Metrics = workflowVersion.getMetricsByPlatform().get(platform1);
+        Metrics platform2Metrics = workflowVersion.getMetricsByPlatform().get(platform2);
+
+        Map<String, Metrics> metricsGet = extendedGa4GhApi.aggregatedMetricsGet(workflowId, workflowVersionId);
```

Comment:
Should this be `Map<Partner, Metrics>`?

Reply:
Hm, yeah it should. Not sure why the endpoint is building it as `Map<String, Metrics> `. I'll take a look

Reply:
Hm, It looks like the Put endpoint also is converting the `Partner` parameter to a String.  https://github.com/dockstore/dockstore/blob/7239b54cdece938b2dca547d70ba58e9e7d02798/dockstore-integration-testing/src/test/java/io/dockstore/client/cli/ExtendedTRSOpenApiIT.java#L137

Reply:
ok, that's probably sufficient for now (for your UI)

Reply:
Looks like OpenAPI defines [enums](https://swagger.io/docs/specification/data-models/enums/) as a string with a range of possible values. Our openapi.yaml also has other enums as strings. I don't think we can control this because it's not possible to have a `Partner` data type for openapi. (Possible datatypes [here](https://swagger.io/docs/specification/data-models/data-types/))